Library/Homebrew/test/test_options.rb

```diff
@@ -56,7 +56,7 @@ def test_equality
 
 class OptionsTests < Homebrew::TestCase
   def setup
-    @options = Options.new
+    @options = Options.create
```

Comment:
* First of all, although `Options` is used to present an array of flags, in order to handle value part for flags like `--with-foo=bar`, the options are stored in a `Hash` internally.
* `Options.create` is used to initialize Options from an array of flags. It will do some preprocess to generate the hash object we will used to present this array of flags.
* `Options.new` will just take hash object directly to create a new options object. This is useful for operators like `+`, `-`, `&`, etc. (See the logic of these methods for example).

>  Will you make new private?

Yes. But how to tag it as private?